tests/unit/test_charm.py

```diff
@@ -369,17 +361,16 @@ def test_given_any_config_and_not_installed_when_on_config_changed_then_status_i
 
         self.assertEqual(self.harness.charm.unit.status, ActiveStatus(""))
 
+    @patch("charm.SrsLteCharm._ue_attached")
     @patch("subprocess.run")
     @patch("builtins.open", new_callable=mock_open)
+    @patch("charm.service_active")
     def test_given_any_config_and_started_is_true_when_on_config_changed_then_srsenb_service_is_restarted(  # noqa: E501
-        self, _, patch_subprocess_run
+        self, _, __, patch_subprocess_run, mock_ue_attached
     ):
         key_values = {}
-        self.harness.update_relation_data(
-            relation_id=self.peer_relation_id,
-            app_or_unit=self.harness.charm.app.name,
-            key_values={"started": json.dumps(True)},
-        )
+        # TODO change when the other PR is merged
```

Comment:
The PR was merged, is this still needed?

Reply:
Fixed